spec/views/application/_header.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "rendering header buttons do" do
+  describe "with no authorized user logged in" do
+    it "has a logout button" do
+      render partial: "application/header.html.erb"
```

Comment:
[praise] I love this approach of testing just the partial to verify this behavior. I always get my view tests way too bogged down in setup by testing the whole page.